### PR TITLE
[bitnami/keycloak] remove Wildfly management port and secret key

### LINKED ISSUE #11129: Keycloak chart still references obsolete Wildfly features
### Name and Version

bitnami/keycloak 9.4.0

### What steps will reproduce the bug?

This issue occurs for any configuration of the chart.

### Are you using any custom parameters or values?

_No response_

### What is the expected behavior?

_No response_

### What do you see instead?

After deploying the chart with the default values, the following resources are created

* A `management` listener for Wildfly management on port 9990 on the pod container
* a management-password in the keycloak secret
  * Leaving this field empty can cause the chart to report
    ```
     PASSWORDS ERROR: You must provide your current passwords when upgrading the release.
     ```
     when an upgrade modifies the keycloak secret (for example, toggling TLS on/off)

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by kherock**
### Description of the change

Since Keycloak 17, there isn't any listener bound to port 9990, and the management password passed to the image isn't used. This PR removes all references to these obsoleted features.

### Benefits

See #11129

### Applicable issues

  - Closes #11129

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Comment by kherock** [comment]
Anything I can do to help move this along?

**Comment by migruiz4** [comment]
I'm sorry for the late response @kherock. 
The PR looks good to me, but could you please fix the branch conflicts and update the chart version at Chart.yaml?

bitnami/keycloak 9.6.1 has already been released

File: bitnami/keycloak/Chart.yaml [hunk 1]
```diff
@@ -26,4 +26,4 @@ name: keycloak
 sources:
   - https://github.com/bitnami/bitnami-docker-keycloak
   - https://github.com/keycloak/keycloak
-version: 9.6.0
+version: 9.6.1
```
Comment (migruiz4) [inline bitnami/keycloak/Chart.yaml:29]:
```suggestion
version: 9.6.2
```

**Review by migruiz4 — CHANGES_REQUESTED** [review]

**Comment by kherock** [comment]
Hey, I think this should be ready!

**Comment by migruiz4** [comment]
Hi @kherock,

The same issue happened again, version 9.6.2 has already been released, could you please rebase your branch and bump another version? 

Please @ me when done.

**Comment by kherock** [comment]
@migruiz4

**Review by migruiz4 — APPROVED** [review]
Looks good to me, thank you for your contribution!

**Pull request merged**